python/architecture.py

```diff
@@ -2394,7 +2393,7 @@ def get_flag_write_low_level_il(self, op, size, write_type, flag, operands, il):
 				operand_list[i].constant = True
 				operand_list[i].value = operands[i]
 		return lowlevelil.LowLevelILExpr(core.BNGetArchitectureFlagWriteLowLevelIL(self.handle, op, size,
-		        self._flag_write_types[write_type], flag, operand_list, len(operand_list), il.handle))
+				self._flag_write_types[write_type], flag, operand_list, len(operand_list), il.handle))
```

Comment:
This looks like it might have been a change from an editor re-aligning? We typically use spaces for indentation for line-wraps intentionally.